164 - Extract camera scanner from core library keeping the same activity (#176)
enioka-Haute-Couture/enioka_scan: docs/dependencies.md

### `scanner`

Core components of the library, **required** for it to work. It includes the scanner APIs, default
scanning activity and service.

### `scanner-camera`

Includes the `CAMERA_SCANNER` provider. It includes camera-as-a-scanner functionality, and

> default camera layout for scanner activity.

Reviewer: ```suggestion
default camera layout for the scanner activity.
```

:widths: auto
:align: center

| Artefact                           | Provider name                    | Supported devices                                              | Tested devices                     | Device type   | External requirements                                          | Device compatible if                                                                                                              |
|------------------------------------|----------------------------------|----------------------------------------------------------------|------------------------------------|---------------|----------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|

> | `core`                             | `CORE` (no provider)             | Does not support any device alone, it required to use the lib. |                                    |               |                                                                |                                                                                                                                   |

Reviewer: ```suggestion
| `core`                             | no provider             | Does not support any device alone, it is a required dependency. |                                    |               |                                                                |                                                                                                                                   |
```

Verifier que le nom d'artefact `core` corresponde bien a l'artefact qui sera publish sur maven (actuellement: `scanner`)

| Artefact                           | Provider name                    | Supported devices                                   | Tested devices                     | Device type   | External requirements                                          | Device compatible if                                                                                                              |
|------------------------------------|----------------------------------|-----------------------------------------------------|------------------------------------|---------------|----------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|

> | `scanner-camera`                   | `CAMERA_SCANNER` (no provider)   | Any device with a camera                            | Smartphones, integrated devices... | Camera        |                                                                | Device has a camera                                                                                                               |

Reviewer: Garder une ligne avec l'artefact "core" en précisant bien qu'il ne supporte aucun SDK par défaut mais est nécessaire a l'utilisation de la lib, et contient notamment le service qui gère la recherche et le lifecycle des scanners (comme la recherche fonctionne par intent, il est il me semble possible de détecter des SDK qui ne sont pas forcement presents dans l'application).
